packages/dashboard-frontend/package.json

```diff
@@ -28,13 +31,36 @@
     "@babel/preset-env": "^7.26.0",
     "@babel/preset-react": "^7.25.9",
     "@jest/globals": "^29.7.0",
+    "@storybook/addon-essentials": "^8.6.12",
+    "@storybook/addon-interactions": "^8.6.12",
```

Comment:
This is for the `play` functionality which is not used in the stories at this point.

Reply:
Added tooltip interactions now